include/z64message.h

```diff
@@ -12,6 +12,17 @@ struct OcarinaStaff;
 extern u16 sBombersNotebookEventMessages[BOMBERS_NOTEBOOK_EVENT_MAX];
 extern u16 gBombersNotebookWeekEventFlags[BOMBERS_NOTEBOOK_EVENT_MAX];
 
+typedef enum FaceReactionSet {
+    /* 0x11 */ FACE_REACTION_SET_JIM = 0x11,
+    /* 0x12 */ FACE_REACTION_SET_BOMBERS,
+    /* 0x15 */ FACE_REACTION_SET_BOMBERS_HIDEOUT_GUARD = 0x15,
+    /* 0x2D */ FACE_REACTION_SET_TREASURE_CHEST_SHOP_GAL = 0x2D,
+    /* 0x2F */ FACE_REACTION_SET_CURIOSITY_SHOP_MAN = 0x2F,
+    /* 0x30 */ FACE_REACTION_SET_TOWN_SHOOTING_GALLERY_MAN,
+    /* 0x31 */ FACE_REACTION_SET_SWAMP_SHOOTING_GALLERY_MAN,
+    /* 0x36 */ FACE_REACTION_SET_PART_TIMER = 0x36
+} FaceReactionSet;
+
```

Comment:
I'm not sure if this is the better header for this stuff, but I think it is fine for now, we can always move it after.

Btw, can you move the function declaration from `functions.h` to this header too?

Reply:
Oh, and can you check if the function would still match using the enum as a type too?
```c
u16 Text_GetFaceReaction(PlayState* play, FaceReactionSet reactionSet);
```

Reply:
>Btw, can you move the function declaration from functions.h to this header too?

No, I can't, at least not without further changes. This function declaration relies on `PlayState` being defined, but `PlayState` is defined in z64.h, which includes `z64message.h`. So as-is, this creates a circular dependency.